api/restHandler/CoreAppRestHandler.go

```diff
@@ -2378,7 +2378,7 @@ func (handler CoreAppRestHandlerImpl) GetAppWorkflowAndOverridesSample(w http.Re
 		common.WriteJsonResp(w, err, nil, http.StatusBadRequest)
 		return
 	}
-	app, err := handler.appCrudOperationService.GetAppMetaInfo(appId)
+	app, err := handler.appCrudOperationService.GetAppMetaInfo(appId, 0, 0)
```

Comment:
We can pass variable name indicating specific ID. and that can be for all the occurrence.

Reply:
updated